packages/expo-store-review/src/ExpoStoreReview.ts

```diff
@@ -1,2 +1,13 @@
-import { NativeModulesProxy } from '@unimodules/core';
-export default NativeModulesProxy.ExpoStoreReview;
+import { Platform } from '@unimodules/core';
+
+export default {
+  get name(): string {
+    return 'ExpoStoreReview';
+  },
+  async isAvailableAsync(): Promise<boolean> {
+    // true on Android, false on web
+    return Platform.OS !== 'web';
```

Comment:
```suggestion
    return Platform.OS === 'android';
```
this would better support future platforms that haven't been implemented yet.